lib/routes/account.js

```diff
@@ -447,15 +518,22 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
-            // The response doesn't have to wait for this,
-            // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
+        function sendVerifyLoginEmail() {
+          // Verify login emails are only sent if the login is requesting keys and the feature is enabled.
+          // In the scenario where keys are requested, but feature is disabled, the keys are
+          // created verified.
+          var shouldSendVerifyLoginEmail = wantsKeys(request) && doSigninConfirmation
+
+          if (shouldSendVerifyLoginEmail) {
+            mailer.sendVerifyLoginEmail(
+              emailRecord,
+              tokenVerificationId,
               userAgent.call({
```

Comment:
@vbudhram - Something about this call to `sendVerifyLoginEmail` or `userAgent` is amiss. The device name is not set in the email:

<img width="365" alt="screen shot 2016-05-24 at 12 45 28" src="https://cloud.githubusercontent.com/assets/848085/15502843/46c44598-21ae-11e6-8381-691a4544b22d.png">

If I click "resend" in the content server, which calls `/recovery_email/resend_code`, the device name is displayed:

<img width="348" alt="screen shot 2016-05-24 at 12 45 35" src="https://cloud.githubusercontent.com/assets/848085/15502851/57a09c7c-21ae-11e6-8a9e-ae5a96f6e4d0.png">